newscoop/classes/NodeTree.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Then please move \Node also to NewscoopBundle. I think that it's used so far only here.